app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -59,7 +59,9 @@ export const FixedSessionsMapCtrl = (
       sensorId,
       location: {address: "", indoorOnly: false, streaming: true},
       tags: "",
-      usernames: ""
+      usernames: "",
+      timeFrom: moment().utc().startOf('day').subtract(1, 'year').format('X'),
+      timeTo: moment().utc().endOf('day').format('X')
```

Comment:
I remember somewhere else we were adjusting the offset after calling `utc()` or something like that. You may want to double check if this or the other should be changed

Reply:
Explanation mostly for documentation purposes: I removed the offset adjusting totally. Now the only place we do the adjusting is here and in the setTimeRangeFilter below. These are the places where we want to make sure that the timezones are not taken into account. Cause in general we just want the time since epoch and we don't care about the time zones during sessions fetching and we don't want to switch time zones while sending links to users in other regions, we want to preserve the same hour.